tools/devctr/Dockerfile

```diff
@@ -147,6 +149,16 @@ RUN mkdir "$TMP_BUILD_DIR" && cd "$TMP_BUILD_DIR" \
     && cd / \
     && rm -rf "$TMP_BUILD_DIR"
 
+# Build iperf3. We need it until 3.18 is released with a fix to a crash on aarch64.
```

Comment:
hmm I hope it will be backported to Ubuntu 24.04 otherwise we will have to keep compiling iperf3